conans/util/tracer.py

```diff
@@ -72,7 +72,7 @@ def _file_document(name, path):
 def log_recipe_upload(ref, duration, files_uploaded, remote_name):
     files_uploaded = files_uploaded or {}
     files_uploaded = [_file_document(name, path) for name, path in files_uploaded.items()]
-    _append_action("UPLOADED_RECIPE", {"_id": str(ref),
+    _append_action("UPLOADED_RECIPE", {"_id": repr(ref.copy_clear_rev()),
```

Comment:
Annotate to add revision info for Conan 2.0. The traces should be complete, including the revision, right?